draft-ietf-mls-architecture.md

```diff
@@ -1019,13 +1019,13 @@ interoperate.
 
 - Additional services may or may not be required depending on the application
   design:
-  - If assisted joining is desired (meaning that the ratchet tree is not
+  - If assisted joining is desired (meaning that the group state is not
     provided in Welcome messages), there must be a method to download the
-    ratchet tree corresponding to a group.
+    group state corresponding to a group.
   - If assisted joining is desired and the Delivery Service is not able to
-    compute the ratchet tree itself (because some proposals or commits are sent
+    compute the group state itself (because some proposals or commits are sent
     encrypted), there must be a method for group members to publish the updated
-    ratchet tree after each commit.
+    group state after each commit.
```

Comment:
I think these cases all refer to the `ratchet_tree` extension, as opposed to the ratchet tree generally. I suggest leaving these as is.